### DISCUSSION TITLE
[PSD10] Exercise 2.39

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#165] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210060627-d8393864-5ec5-4d8b-a78c-da249d4a5bcb.png)

**Comment by Elia-Belli** [comment]
L'output è 1 quando A=1 o quando A=0 e il primo mux seleziona 00 o 11: 
$Y= A + \bar A \bar C \bar D + \bar A C D$
$A + \bar A (CD+ \bar C \bar D)$
$(A + \bar A)(A+CD + \bar C \bar D)$
$A+CD + \bar C \bar D$

**Reply by neoSnakex34** [reply]
mi torna,le soluzioni ufficiali considerano anche (come equivalente) y = a + !(c xor c)

**Comment by fraxb04** [comment]
errato, occorre applicare prima la distributiva (raccogliendo !A!D) e poi l'assorbimento (tra !A!D e A), quindi la semplificazione dell'equazione sarebbe A + !D

**Comment by Elia-Belli** [comment]
A me sembra corretto, ho esplicitato alcuni passaggi, forse era poco chiaro

**Comment by fraxb04** [comment]
scusami, mi sono accorto ora che ho disegnato male il mux (ho messo al posto di 10, 11)....

quindi è corretto
